packages/lab/src/date-picker/DatePicker.tsx

```diff
@@ -275,7 +277,7 @@ export const DatePicker = forwardRef<
         ref={floatingRef}
         {...getFloatingProps()}
         onSelect={handleSelect}
-        CalendarProps={CalendarProps}
+        CalendarProps={{ ...CalendarProps, borderedDropdown: bordered }}
```

Comment:
FYI: you can't override it with CalendarProps, will fix in refactor, no need to change in this PR